pages/api/logout.ts

```diff
@@ -0,0 +1,14 @@
+import useUser from '@lib/useUser'
+import { withIronSessionApiRoute } from 'iron-session/next'
+import { sessionOptions } from 'lib/session'
+import { NextApiRequest, NextApiResponse } from 'next'
+import type { User } from 'pages/api/user'
+
+export default withIronSessionApiRoute(logoutRoute, sessionOptions)
+
+function logoutRoute(req: NextApiRequest, res: NextApiResponse<User>) {
+  req.session.destroy()
+  //res.json({ isLoggedIn: false, login: '', avatarUrl: '' })
+  // TODO
+  res.redirect('/login');
```

Comment:
Det här känns rätt osäkert, då vi bara gör en fetch från klienten. Redirecten gör client side när den här requesten lyckats

Reply:
This is a todo, logout exists in a seperate issue